brainzutils/musicbrainz_db/event.py

```diff
@@ -0,0 +1,100 @@
+from collections import defaultdict
+from mbdata import models
+from brainzutils.musicbrainz_db import mb_session
+from brainzutils.musicbrainz_db.utils import get_entities_by_gids
+from brainzutils.musicbrainz_db.includes import check_includes
+from brainzutils.musicbrainz_db.serialize import serialize_events
+from brainzutils.musicbrainz_db.helpers import get_relationship_info
+
+
+def get_event_by_id(mbid, includes=None):
+    """Get event with the MusicBrainz ID.
+
+    Args:
+        mbid (uuid): MBID(gid) of the event.
+    Returns:
+        Dictionary containing the event information.
+    """
+    if includes is None:
+        includes = []
+
+    return _get_event_by_id(
+        mbid,
+        includes=includes
+    )
+
+
+def _get_event_by_id(mbid, includes=None):
```

Comment:
I guess we don't need these separate functions like `_get_event_by_id`. These functions are very similar to functions which call them, for example `get_event_by_id`. In CB code, there were two separate functions as caching was involved.